src/compiler/types.rs

```diff
@@ -954,6 +1014,15 @@ impl Symbol {
     }
 }
 
+pub struct InternalSymbolName;
+
+#[allow(non_snake_case)]
+impl InternalSymbolName {
+    pub fn Object() -> __String {
+        __String::new("__object".to_string())
```

Comment:
`InternalSymbolName` is a string-valued enum in the Typescript version (where the values are treated as the special `__String` type). I didn't see a way that the pattern of grabbing these known `__String`'s wouldn't currently involve copying them (since the things consuming this are expecting to own a `__String`) so just modeled this as getters that return a new `__String`